lib/pbench/test/unit/server/conftest.py

```diff
@@ -729,6 +729,21 @@ def pbench_token_invalid(client, create_drb_user):
     return generate_token(username="drb", user=create_drb_user, valid=False)
 
 
+@pytest.fixture()
+def get_token(pbench_admin_token):
+    """
+    Get OIDC token for any user specified by the username.
+    """
+
+    def token(username: str) -> str:
```

Comment:
The docstring at line 735 is really the docstring for the inner function (at line 738).

For `get_token()` I would be inclined to be specific and explicit:
> This fixture yields a function value which can be called to get an OIDC token for the user corresponding to the specified username.

BTW, [PEP-8](https://peps.python.org/pep-0008/#documentation-strings) recommends putting the opening line of the docstring on the same line as the opening triple-quote, and it suggests including the closing triple-quote on the same line for single-line docstrings (subject to line-length constraints, obviously).

Reply:
> BTW, [PEP-8](https://peps.python.org/pep-0008/#documentation-strings) recommends putting the opening line of the docstring on the same line as the opening triple-quote

Yeah, I've always found that style ugly and extremely awkward, but I've been trying to get used to it. 🍤

Reply:
Yeah, me too.  😕  But, if we're gonna follow PEP-8, we should follow PEP-8.  (Although, I did note that Black is willing to negotiate on a couple of details, here.  😇)